packages/core/src/Chip/Chip.spec.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react'
 import { fireEvent, render, screen } from 'testing-library'
 import { act, renderHook } from '@testing-library/react-hooks'
+import { theme } from '..'
```

Comment:
non-blocking, so this doesn't have to be updated in the future :P

```suggestion
import { createTheme } from '..'

const theme = createTheme()
```